packages/nuxt/src/vite/sourceMaps.ts

```diff
@@ -0,0 +1,33 @@
+import type { Nuxt } from '@nuxt/schema';
+import { sentryVitePlugin } from '@sentry/vite-plugin';
+import type { SentryNuxtModuleOptions } from '../common/types';
+
+/**
+ *  Setup source maps for Sentry inside the Nuxt module during build time.
+ */
+export function setupSourceMaps(moduleOptions: SentryNuxtModuleOptions, nuxt: Nuxt): void {
+  nuxt.hook('vite:extendConfig', async (viteInlineConfig, _env) => {
+    const sourceMapsUploadOptions = moduleOptions.sourceMapsUploadOptions || {};
+
+    if ((sourceMapsUploadOptions.enabled ?? true) && viteInlineConfig.mode !== 'development') {
+      const sentryPlugins = sentryVitePlugin({
+        org: sourceMapsUploadOptions.org ?? process.env.SENTRY_ORG,
```

Comment:
l: Do we need to pass in the env variable fallbacks explicitly? the plugin itself should pick them up, right?